org/teams.yaml

```diff
@@ -237,6 +237,7 @@ teams:
           - kyessenov
           - mandarjog
```

Comment:
we should probably remove `mandarjog` and `gargnupur` from this list at the same time.